pkg/operations/cordondrainvm.go

```diff
@@ -135,7 +136,8 @@ func daemonSetPodFilter(pod v1.Pod) bool {
 // getPodsForDeletion returns all the pods we're going to delete.  If there are
 // any pods preventing us from deleting, we return that list in an error.
 func (o *drainOperation) getPodsForDeletion() (pods []v1.Pod, err error) {
-	podList, err := o.client.ListPods(o.node)
+	podList, err := o.client.ListPods(metav1.NamespaceAll, metav1.ListOptions{
```

Comment:
This change is owing to a change in the k8s client-go version?

Reply:
Since this is an Exportable interface that we maintain, we shouldn't change its method signature. Probably should add a new ListPodsByNameSpace (or something) method